src/Http/ViewComposers/ConfigComposer.php

```diff
@@ -44,7 +44,24 @@ public function compose(View $view)
         Config::set('frontend.show_tax', (bool) Rapidez::config('tax/display/type', 1));
         Config::set('frontend.grid_per_page', Rapidez::config('catalog/frontend/grid_per_page', 12));
         Config::set('frontend.grid_per_page_values', explode(',', Rapidez::config('catalog/frontend/grid_per_page_values', '12,24,36')));
-        Config::set('frontend.queries.cart', view('rapidez::cart.queries.cart')->renderOneliner());
+        // Not sure if we should continue this way. All above is pretty repeated.
+        // We could assign everything as nested array at once but it could make
+        // it harder to override if there was already something set from a
+        // package or project? Not sure what the order will be.
```

Comment:
To be fair, all current values are read from Magento config.
If someone wanted to override this they should change the Magento config.

I think it's safe to put this in an array, and fire an event in case people actually want to override these values